File: GameStats.React/ClientApp/src/DTOs/Team/TeamDto.js
export default class TeamDto {
    constructor(options = {}) {
        this.id = options.id || 0;
        this.name = options.name || '';
        this.leagueId = options.leagueId || 0;
        this.seasonId = options.seasonId || 0;
        this.leagueDescription = options.leagueDescription || '';
        this.seasonDescription = options.seasonDescription || '';
        
    }

    stringify() {
        return JSON.stringify(this);
    }
}

File: GameStats.Common/Contracts/Team/TeamDto.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace GameStats.Common.Contracts.Team
{
    public class TeamDto
    {
        [JsonPropertyName("id")]
        public int Id { get; set; }

        [JsonPropertyName("name")]
        public string Name { get; set; }

        [JsonPropertyName("leagueId")]
        public int LeagueId { get; set; }

        [JsonPropertyName("seasonId")]
        public int SeasonId { get; set; }
        
        [JsonPropertyName("leagueDescription")]
        public string LeagueDescription { get; set; }

        [JsonPropertyName("seasonDescription")]
        public string SeasonDescription { get; set; }
    }

    public class TeamListDto
    {
        public List<TeamDto> Teams { get; set; } = new();
    }
}
